src/scripts/main.js

```diff
@@ -1 +1,44 @@
 'use strict';
+
+const form = document.getElementById('form');
+form.addEventListener('submit',function(event){
+  event.preventDefault();
+
+  window.scrollTo({
+    top:0,
+    behavior:'smooth'
+  })
+})
+
+
+
+  document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I understand your idea here, but I will give you a suggestion that could improve on visual:
Because you are using a class "active" to select what image is showing, when using prev and next, the image "jumps" changing without an animation. that isn´t a real problem, but everything else on the website uses animation, and this this looks sorta out of place.
Instead of using the active class, you could use currentIndex to `transform: translateX(${currentIndex * 100%)` in the slider. To have a sliding animation when changing images.

That, of course, is not mandatory, just an idea if you want to implement it